baseplate/lib/message_queue.py

```diff
@@ -166,7 +249,12 @@ def queue_tool() -> None:
 
     args = parser.parse_args()
 
-    queue = MessageQueue(args.queue_name, args.max_messages, args.max_message_size)
+    if args.use_in_memory_queue:
```

Comment:
I think Fred's queue type idea is better here with a sensible default if not set.